task/mgmt-gateway/src/main.rs

```diff
@@ -0,0 +1,466 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#![no_std]
+#![no_main]
+
+use drv_stm32h7_usart::Usart;
+use gateway_messages::sp_impl;
+use gateway_messages::sp_impl::Error as MgsDispatchError;
```

Comment:
Hah, this is an interesting difference. I have my editor configured specifically _not_ to use collapsing via braces because that style in omicron was tending to cause annoying merge conflicts (in files with many imports that were also often modified concurrently). I don't have a strong opinion on this and am happy to add curly braces if that's more consistent with hubris.

Reply:
Changed in 60ec1d5e